tests/ha/multi_pod_failure/test_multi_data_pod_failure.py

```diff
@@ -2535,16 +2329,16 @@ def test_degraded_mpu_after_kpods_fail(self):
                                   f"Failed to match checksum: {upload_checksum},"
                                   f" {download_checksum}")
         LOGGER.info("Matched checksum: %s, %s", upload_checksum, download_checksum)
-        LOGGER.info("Step 6: Successfully downloaded the object and verified the checksum")
+        LOGGER.info("Step 3: Successfully downloaded the object and verified the checksum")
 
         LOGGER.info("Removing files %s and %s", self.multipart_obj_path, download_path)
         sysutils.remove_file(self.multipart_obj_path)
         sysutils.remove_file(download_path)
 
-        LOGGER.info("Step 7: Create new bucket again and do multipart upload. "
+        LOGGER.info("Step 4: Create new bucket again and do multipart upload. "
                     "Download the object & verify checksum.")
-        bucket_name = "mp-bkt-{}".format(self.random_time)
-        object_name = "mp-obj-{}".format(self.random_time)
+        bucket_name = f"mp-bkt-{int(perf_counter_ns())}"
+        object_name = f"mp-obj-{int(perf_counter_ns())}"
```

Comment:
Are you taking care of DTM changes in this?

Reply:
> Are you taking care of DTM changes in this?

Yes. I have taken care most of the places. But once all changes are done will take a final look to take care of missing DTM or s3bench installation flag to False.

Reply:
So if you are providing new bucket for mpu, that bucket will be created. You need to add DTM condition here.